manifests/non_chroot.pp

```diff
@@ -37,15 +33,6 @@
     require => Package['bind']
   }
 
-  file { '/var/named':
```

Comment:
When you moved this to init.pp, you removed the notify and require.  Why?

Reply:
@lnemsick-simp They're handled by `autorequires` in the various components and the Package/Dir relationship doesn't matter in terms of order.